indico/MaKaC/participant.py

```diff
@@ -448,16 +448,8 @@ def addPendingParticipant(self, participant):
 
             profileURL = urlHandlers.UHConfModifParticipantsPending.getURL(self._conference)
 
-            toList = []
-            creator=self._conference.getCreator()
-            if isinstance(creator, AvatarUserWrapper):
-                toList.append(creator.getEmail())
-            for manager in self._conference.getAccessController().getModifierList() :
-                if isinstance(manager, AvatarUserWrapper):
-                    toList.append(manager.getEmail())
-
             data = {}
-            data["toList"] = toList
+            data["toList"] = self._conference.all_manager_emails
```

Comment:
When emailing all managers I think having them all in To is not that bad - it's not likely to be a completely unrelated bunch of people and being able to reply-all on such emails sounds somewhat useful